src/netlify.toml

```diff
@@ -0,0 +1,4 @@
+[build]
+base = "/"
+publish = "/public"
```

Comment:
is there a `/public` directory?

Reply:
just realized that the config file is not in the root dir